src/decoder.rs

```diff
@@ -94,12 +200,12 @@ mod tests {
         }
 
         #[test]
-        fn test_example_6() {
+        fn example_6() {
             assert_eq!(decode("=?ISO-8859-1?Q?a_b?=").unwrap(), "a b");
         }
 
         #[test]
-        fn test_example_7() {
+        fn example_7() {
```

Comment:
Could be a good occasion to name those tests properly (with a more explicit name).

Reply:
done